frame/scored-pool/src/lib.rs

```diff
@@ -222,11 +222,27 @@ decl_event!(
 	}
 );
 
+decl_error! {
+	/// Error for the scored-pool module.
+	pub enum Error for Module<T: Trait<I>, I: Instance> {
+		/// Already a member.
+		AlreadyInPool,
+		/// Balance is too low to submit candidacy.
+		InsufficientBalance,
```

Comment:
Do we need this error?

Balances module already has one

Reply:
Nice catch, though `InsufficientBalance` from balances is not in the master yet. :wink: 
The test in `scored-pool` should be updated too, so I'm going to fix this after your pull request is merged.